cv32/tests/programs/custom/cv32e40p_csr_access_test/cv32e40p_csr_access_test.S

```diff
@@ -22,6 +22,12 @@ _start_main:
 #
 ############################################
 
+	#ifdef NO_PULP
```

Comment:
There is going to be a significant conflict here as I have made substantial changes to this test-program.   Having said that, I do like the sentiment of these changes, so I will incorporate them into my version.

Reply:
Thanks.  I wasn't sure how these were generated.